app-admin/locksmith/locksmith-9999.ebuild

```diff
@@ -23,20 +23,18 @@ LICENSE="Apache-2.0"
 SLOT="0"
 IUSE=""
 
-DEPEND=">=dev-lang/go-1.2"
-
 src_compile() {
 	# work around gentoo hardened gcc incompatibilities with cgo
 	# see https://bugs.gentoo.org/show_bug.cgi?id=493328
 	if gcc-specs-pie; then
-		GOLDFLAGS="-extldflags -fno-PIC"
+		CGO_LDFLAGS+=" -extldflags -fno-PIC"
```

Comment:
Yeah, I think it was just that no PIC implies no PIE but itself PIC doesn't actually matter one way or the other because the final product is an executable, not a library.